src/assets/mixins.styl

```diff
@@ -0,0 +1,16 @@
+//fonts mixins
+play($weight)
+  font-family 'Playfair Display', serif
+  font-weight $weight
+
+
+source($weight)
```

Comment:
change the name of this mixin, is not very descriptive, also check that the previous mixin is almost the same except for the font so have in mind one of our most important principles DRY(Don't repeat yourself)